### DISCUSSION TITLE
4.0.5 on macOS Sonoma 14.1.2 - low peer connections

### DISCUSSION THREAD

**Discussion by rajopotamus** [transmission/transmission#6369] [Q&A]
Hello everyone - I know this is a topic that's come up often; however I can't for the life of me figure out why my peer count is so low when using transmission 4.0.x.   When going to 3.0 it's better but not great compared to using something like qbit.  It seems the connected number of peers is lower, and thus the overall file download speed is much lower.  Im not sure if anyone else is having these problems.  qbit for Mac is very unstable, but it's very fast when downloading the same torrent.  Any insight on what I can do to try and resolve this issue?

**Comment by dantearmok** [comment]
What are your speed'n'peer type settings?

**Comment by rajopotamus** [comment]
Good Evening Dan - 

I attached some screen shots of my settings.  When I use these same exact settings on transmission 3.0; performance improves but still no where near where it is with other clients; just improves relatively.   It seems that on 4.0.5 my total download speed peaks around 15-20 mb/s.  On 3.0 it’s around 35-50 mb/s.   On qbit it’s 90-100 mb/s.   I have a gigabit fios connection.  Thank you for reaching out to me and let me know what your thoughts are.  Have a wonderful evening.  

Sincerely, 
Raj 


> On Dec 12, 2023, at 12:35 AM, Dan ***@***.***> wrote:
> 
> 
> What are your speed'n'peer type settings?
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/transmission/transmission/discussions/6369#discussioncomment-7826859>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BD5ESEU66H7EB2YSCBR3NB3YI7UJ3AVCNFSM6AAAAABAQXLFAGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQMRWHA2TS>.
> You are receiving this because you authored the thread.
> 
￼￼￼

**Reply by arcticfoxwp** [reply]
This is a long-standing issue that still persists in the latest nightlies.

Please post your findings here:

https://github.com/transmission/transmission/issues/5261
